_positions/site-reliability-engineer.md

```diff
@@ -32,3 +32,5 @@ Site Reliability Engineers ensure our platforms and infrastructure systems, as w
 - Support and empathize with development teams by working closely with them to manage system lifecycles, including change and configuration management.
 - Create tooling to reduce developer workload in deploying and managing their applications.
 - Evangelize the importance of the platform.
+
+##[Apply to join 18F]({{ site.baseurl }}/pages/apply.html)
```

Comment:
Same questions as in #207/209. (No need to answer in each PR, just in #207 or #209.)